gitvs/src/components/GetProfile.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import './getProfile.css'
+import LogoUrlChanger from './LogoUrlChanger'
+
+function GetProfile({name, avatar_url, public_repo, location, weapons}) {
+    return (
+        <div className='card'>
+            <h1 className='githubNickname'>{name}</h1>
+                <div className='avatarBox'>
+                    <img className='avatar' src={avatar_url} alt='avatar'></img>
```

Comment:
You can use the **name** in **alt** to have a better description

Reply:
Fixed